packages/plugin-infra/src/web3/utils/isSameAddress.ts

```diff
@@ -0,0 +1,4 @@
+export function isSameAddress(a = '', b = '') {
```

Comment:
Please read this from `Utils.isSameAddress`.

Reply:
done